ICP-5091 Enable Tamper detection by default for Fibaro CO Sensor

greens opened the pull request.

greens committed:
ICP-5091 Enable Tamper detection by default for Fibaro Smoke Sensor

dkirker reviewed (dismissed):
It looks good. There's the chance that the initial syncStart might not run because it is called from updated(), however there is syncNext which gets called from the WakeUpNotification, so I think that will be ok.

greens commented:
updated

greens committed:
ICP-5091 Enable Tamper Detection by default for Fibaro CO sensor

Had to add code changes since we now do want to send at least one configuration set on add.
